qiling/extensions/tracing/formats/registers.py

```diff
@@ -0,0 +1,111 @@
+import unicorn
+from qiling.arch import arm64, arm, x86
+
+
+
+class ArchRegs():
+    """
+    Stores architecture's registers relevant to tracing in a format
+    universally usable by different tracers
+    """
+
+    arm_registers = {
```

Comment:
These registers mappings are [somewhat] a duplication of the arch classes registers mappings.
Can we re-use the existing ones to create these and avoid code duplication? (with some necessary modifications, of course)